### ISSUE TITLE
[QA Feedback] "Get Started" modal for Klima Infinity

### ISSUE THREAD

**Issue by ShimonD-KlimaDAO** [KlimaDAO/klimadao#693]
1. There is no light mode in the KI landing page yet, so light mode can't be tested.
![image](https://user-images.githubusercontent.com/99001851/189452025-899be568-cb84-4aca-aa68-7e748cbbbb70.png)

Figma:
![image](https://user-images.githubusercontent.com/99001851/189452095-199333b2-561d-4529-b783-5406faeeb73e.png)

2. "Get Started" button at the bottom of the page doesn't open the new modal, and leads [here](https://www.klimadao.finance/blog/klima-infinity-user-guide). This is consistent with the figma, but it seems logically inconsistent. Just wanted to point that out in case it's an oversight.

3. The modal is not popping up at all on Microsoft Edge.

4. "GET STARTED" at the top of the modal is not supposed to be capitalized according to the figma.
![image](https://user-images.githubusercontent.com/99001851/189452939-fbbee0cf-5754-492c-92bd-264efa9252ac.png)

**Comment by Atmosfearful** [comment]
Thanks @ShimonD-KlimaDAO for #1 we decided we will never support light mode on this page, it made designer life too difficult.

2,3,4 do seem like valid issues, good catch!

**Comment by ShimonD-KlimaDAO** [comment]
#1 is irrelevant.
#2 is resolved.
#3 and #4 are still issues. On Edge, my popup blocker is fully disabled and yet the modal doesn't come up.

**Comment by jabby09** [comment]
@goatsOnDrugs see shimon's QA feedback above

**Comment by jabby09** [comment]
@goatsOnDrugs also want to change the context / text on the modal.

From "I'm not sure which option is for me."
to "I'd like assistance with my organization's carbon footprint."

**Comment by goatsOnDrugs** [comment]
@ShimonD-KlimaDAO 

3. Unable to reproduce on edge + windows 11. What windows version did you see this issue on?
4. The titles for all modals are now capitalized instead of uppercase based on suggestion by @theprofessora 
5. @jabby09 the "not sure" copy has been updated. @theprofessora I had to bump the font size down to avoid the text going on two lines and causing the user to need to scroll slightly.
